composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "hootlex/laravel-moderation",
+  "name": "nguyentranchung/laravel-moderation",
```

Comment:
Looks like this was changed by mistake?

Reply:
Sorry about mistake, but allow edits from maintainers was enabled.